Homework070616.sql

```diff
@@ -0,0 +1,147 @@
+--Get the average rating for a movie
+
+SELECT movies.title, avg(ratings.rating)
+FROM movies
+  JOIN ratings ON movies.movieid = ratings.movieid
+GROUP BY movies.title;
+
+--Get the total ratings for a movie
+
+SELECT movies.title, count(ratings.rating)
+FROM movies
+  JOIN ratings ON movies.movieid = ratings.movieid
+GROUP BY movies.title;
+
+--Get the total movies for a genre
+
+SELECT genre.genres, count(movies.title)
+FROM genre
+  JOIN movie_genre ON genre.id = movie_genre.genre_id
+  JOIN movies ON movie_genre.movieid = movies.movieid
+GROUP BY genre.genres;
+
+--Get the average rating for a user
+
+SELECT ratings.userid, avg(ratings.rating)
+FROM ratings
+GROUP BY ratings.userid;
+
+--Find the user with the most ratings
+
+SELECT ratings.userid, count(ratings.rating)
```

Comment:
As a note from class you can alias the field with `count(ratings.rating) as rating_count`